### PR TITLE
Added server_time function to check server time from Binance API

### PR DESCRIPTION
**Pull request by eugenekartvelishvili**
### Pull request checklist

- [x] Did you bump the project version?
- [x] Did you add a description to the Pull Request?
- [x] Did you add new tests?
- [x] Did you add reviewers?

## Description

Added the `server_time` function to check the server time from Binance API.

## Changes

- Added `ServerTimeQuery` and `ServerTimeData` structs.
- Implemented `server_time` function.
- Updated project version to 0.25.0 in `Project.toml`.
- Removed unnecessary blank line in `Spot.jl`.
- Updated `ServerTimeData` to use `NanoDate` instead of `Int64`.
- Separated exports into lines.
- Updated the method declaration.
- Updated docs for `server_time`
- Updated comments for server_time

### CODE REVIEW

File: src/Binance/Spot/Spot.jl [hunk 1]
```diff
@@ -35,4 +35,8 @@ using .Ticker
 include("API/WithdrawalLog.jl")
 using .WithdrawalLog
 
+include("API/ServerTime.jl")
+using .ServerTime
+
+
```
Comment (gryumov) [inline src/Binance/Spot/Spot.jl:41]:
```suggestion
```

File: src/Binance/Spot/API/ServerTime.jl [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Binance
+using CryptoAPIs: Maybe, APIsRequest
+
+struct ServerTimeQuery <: BinancePublicQuery end
+
+struct ServerTimeData <: BinanceData
+    serverTime::Maybe{Int64}
```
Comment (gryumov) [inline src/Binance/Spot/API/ServerTime.jl:14]: Should it definitely be Maybe{Int64}?

**Comment by gryumov** [comment]
Did you bump the project version?

File: src/Binance/Spot/API/ServerTime.jl [hunk 3]
```diff
@@ -0,0 +1,54 @@
+module ServerTime
+
+export ServerTimeQuery, ServerTimeData, server_time
```
Comment (yshadrin) [inline src/Binance/Spot/API/ServerTime.jl:3]: It is better to separate them into lines

Comment (yshadrin) [reply src/Binance/Spot/API/ServerTime.jl:3]:
```suggestion
export ServerTimeQuery,
    ServerTimeData, 
    server_time
```

File: src/Binance/Spot/API/ServerTime.jl [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Binance
+using CryptoAPIs: Maybe, APIsRequest
+
+struct ServerTimeQuery <: BinancePublicQuery end
```
Comment (yshadrin) [inline src/Binance/Spot/API/ServerTime.jl:11]:
```suggestion
Base.@kwdef struct ServerTimeQuery <: BinancePublicQuery
    #__ empty
end
```

File: src/Binance/Spot/API/ServerTime.jl [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+}
+```
+"""
+function server_time(client::BinanceClient)
+    return APIsRequest{ServerTimeData}("GET", "api/v3/time", ServerTimeQuery())(client)
+end
+    
+function server_time(; client::BinanceClient = Binance.Spot.public_client)
+    return server_time(client)
+end
```
Comment (yshadrin) [inline src/Binance/Spot/API/ServerTime.jl:52]: organize the method declaration in the same way as it is implemented [here](https://github.com/bhftbootcamp/CryptoAPIs.jl/blob/master/src/Binance/USDMFutures/API/ExchangeInfo.jl)

**Comment by yshadrin** [comment]
It is also necessary to correct the documentation according to this [guide](https://github.com/bhftbootcamp/CryptoAPIs.jl/blob/master/src/Binance/USDMFutures/API/ExchangeInfo.jl)

**Review by gryumov — APPROVED** [review]

**Pull request merged**